tests/helpers.py

```diff
@@ -0,0 +1,49 @@
+from typing import Any, Optional
+from core.schemas import observable
+import unittest
+from core.database_arango import ArangoYetiConnector
+
+
+class TestHelpers(unittest.TestCase):
```

Comment:
Maybe rename this class to something a bit more intuitive (e.g. `YetiTestCase`?)